### DISCUSSION TITLE
Plans for my future (modest) works for Elexis

### DISCUSSION THREAD

**Discussion by ngiger** [elexis/elexis.github.io#3] [Ideas]
Hi everybody
I am back from my 4 months long bike trip to the Nordkap and back to Switzerland and enjoying my retirement and my family.
I decided to spend about one day per week to advance some work I never finished for Elexis.
The following decisions will guide my work.

* Bruno Büchel wants to update his french Elexis and become a Medelexis customer to be able to use some custom plugins and agreed to sponsor the french version with some money.
* Victor Mboka will do all the support for Bruno and once he is able to work he would like to adapt the french version of Elexis for his country of origin Congo

My contributions could be like this
* Make Elexis work for Bruno Büchel in Yverdon
  * Migration of his old 2.17 database
  * Fix errors which make french version unusable
  * Update french translation for core, base and medelexis plugins used by Bruno
* Make french/english version of Artikelstamm work
* French translation for rest of plugins
* Automatic migration of a mysql database to postgres

Please answer (here or directly to me)  if you have an interest to pursue any of these points

I do NOT intend to pursue any work with automated functional testing, as it gives me too much (unpaid) work. Would only continue if I get another collaborator helping me. First step would be to decide whether it is worth continuing with rcptt or whether we should switch to a more healthy project. But I do not know any open source testing projects, which has an ongoing activity and development. SWTBot is closer to it. RCPTT is in deep maintenance. No recent Jubula version. 

Best regards
Niklaus Giger

**Comment by col-panic** [comment]
Good to see you're back! If we find enought interested people, then we might even revive the 
Elexis Developers meeting!

The Hangout time is still the same - so feel free to join anytime you want to discuss details! 🤗
